app/views.py

```diff
@@ -0,0 +1,11 @@
+from flask_wtf import FlaskForm
+from wtforms import SubmitField
+
+class ConfirmationForm(FlaskForm):
+    submit = SubmitField('Do it')
+
+class ConfirmationFormData(object):
+    def __init__(self, title=None, action=None, backTo=None):
+        self.title = title
+        self.action = action
+        self.backTo = backTo
```

Comment:
I guess it would be best to merge #34 first, and if you decide to go the decorator way, we could use the decorator here too.